### ISSUE TITLE
Custom Sidebar Issue

### ISSUE THREAD

**Issue by tderosier** [slightlyoffbeat/brew#19]
When I followed the example of creating a new sidebar, everything works great except it doesn't format currently when viewing it on a mobile device. It doesn't pull to the left side completely and the margins/padding is off. It's like it's not pulling in the CSS correctly. I'm sure it's an easy fix, but I can't seem to figure it out.

**Comment by slightlyoffbeat** [comment]
is it possible to see the page you are working on?

**Comment by tderosier** [comment]
Hey Dan, sorry I ended up figuring out it was as issue with a certain widget/plugin and not from the Brew Framework itself. I believe everything is fine now. 

Quick question though, is there a correct way to use font awesome icons within the nav menu? What I'm doing is adding the proper css to the menu item name + text. Everything looks great visually, but when I mouse over the menu name, the css text appears. I have tried adding in text for the label on the menu item but that doesn't seem to resolve it either. Any ideas? See images link below: 

https://dl.dropboxusercontent.com/u/68870/cart_mouseover.jpg
https://dl.dropboxusercontent.com/u/68870/menu.PNG

**Comment by slightlyoffbeat** [comment]
absolutely!  In the Title-attribute text area, just include the classes needed.  So just type fa fa-shopping-cart

https://www.dropbox.com/s/jzk327t3lqyljfx/Screenshot%202014-02-05%2014.03.06.png

**Comment by tderosier** [comment]
Haha Duh .. I don't know why I didn't try that. Thanks for the help!

**Comment by slightlyoffbeat** [comment]
no prob :)

**Comment by tderosier** [comment]
I ran into another sidebar issue and I figured out that the addition of "pull-right" in the sidebar.php was causing issues when viewing on a mobile device. Basically the sidebar was not resizing to the full width. After taking out "pull-right", everything works fine. Was there a reason this was added in in the first place?

**Comment by slightlyoffbeat** [comment]
I've not seen that issue, however you are correct, there is no need for "pull-right".  I'll remove it.

**Comment by tderosier** [comment]
It seems to only happen to me when using certain widget plugins. Thanks!

Also, I did notice that in the smaller mobile view, the sidebar wrapper is not the same as the main content wrapper. The sidebar extends out a bit wider.

**Comment by slightlyoffbeat** [comment]
Do you have a screenshot?

**Comment by tderosier** [comment]
Here's a screenshot from my ipod. See how the sidebar area extends further: https://www.dropbox.com/s/llqhergi65kbc7y/photo.PNG

**Comment by tderosier** [comment]
And here's what it looks like when I change the sidebar background to white: https://www.dropbox.com/s/acic6aft9wqzamt/photo2.PNG
